trex/gauges.go

```diff
@@ -8,6 +8,11 @@ import (
 var nameSpace = "trex"
 
 var (
+	Uptime = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Would you lowercase? sugged `uptimeGauge`

Reply:
Done